stdlib/public/core/StringBridge.swift

```diff
@@ -93,6 +93,33 @@ internal func _cocoaHashASCIIBytes(
   return _swift_stdlib_CFStringHashCString(bytes, length)
 }
 
+// These "trampolines" are effectively objc_msgSend_super.
+// They bypass our implementations to use NSString's
+
+@_effects(readonly)
+internal func _cocoaCStringUsingEncodingTrampoline(
+  _ string: _CocoaString,
+  _ encoding: UInt)
+  -> UnsafePointer<UInt8>? {
+    return _swift_stdlib_NSStringCStringUsingEncodingTrampoline(
+      string,
+      encoding)
+}
+
```

Comment:
stdlib formatting:

```swift
@_effects(readonly)
internal func _cocoaCStringUsingEncodingTrampoline(
  _ string: _CocoaString,
  _ encoding: UInt
) -> UnsafePointer<UInt8>? {
...
}
```

Reply:
Do we have a style guide somewhere that I missed? Sorry!

Reply:
No, @moiseev was talking about adding that to the stdlib progman last year. I think once we have better tooling, we'll just swift-format everything and document the style.